### PR TITLE
adding django files

### PR DESCRIPTION
**Pull request by xraycodes**
adding django files onto main

### CODE REVIEW

**Comment by LTS42** [comment]
`__pycache__` folder should not be checked in   
search how you can use `.gitignore` file to ignore unwanted files not to be checked in to git.

I'll continue to review after this has been fixed.

**Comment by LTS42** [comment]
please include how I can install and run this project in a readMe.

**Comment by LTS42** [comment]
think about using `venv` or `virtualenv`

**Comment by LTS42** [comment]
refer to a good example [here](https://github.com/gothinkster/django-realworld-example-app)
briefly look at python environment managers [here](https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe) -- always go with officially supported one (venv)

**Comment by xraycodes** [comment]
> think about using `venv` or `virtualenv`



> please include how I can install and run this project in a readMe.

is there a way to push requirment.txt and readme.txt? I just have these 2 files on desktop. Do I just create a new folder, put these files in there, clone the repository and push it from there?

**Comment by LTS42** [comment]
It's good that you added the two files.
I still see `pycache` folders. Can you attempt to delete them?
I think the command was `git rm --cached file_to_remove` see [link](https://www.git-tower.com/learn/git/commands/git-rm)

File: requirments.txt [hunk 1]
```diff
@@ -0,0 +1,74 @@
+asgiref==3.5.2
```
Comment (LTS42) [inline requirments.txt]: are you sure we absolutely need ALL of these software to run a django server that has simple db connection to sqlite3?

Comment (xraycodes) [reply requirments.txt]: The first requirement I tried using pip freeze > requirements but that may have saved some extra unnecessary packages.

So I installed pipreqs and that was able to shorten the requirements. It should work.  Pushed another requirement.txt

Comment (xraycodes) [reply requirments.txt]: all the pyc. files deleted and pushed the project

Comment (LTS42) [reply requirments.txt]: good. I'll run the project and review the code.
you can expect comments tmr.

**Comment by LTS42** [comment]
I was able to run the server just fine.  
Since the code is all boilersplate code, there were little to review, I've approved
Good job!

Capture

You can now merge and start another pull request based off of the `main` branch

**Review by LTS42 — APPROVED** [review]

**Comment by LTS42** [comment]
This PR has been approved. You can merge and move on to the next 👍

**Pull request merged**